server/services/offlinePayment.service.js

```diff
@@ -42,17 +37,26 @@ export const addOfflinePayment = async (offlinePayment) => {
     );
   }
 
-  const property = await getPropertyById(offer.propertyId);
-
   try {
     const payment = await new OfflinePayment(offlinePayment).save();
-    const description = `You added an offline payment of ${getMoneyFormat(
-      payment.amount,
-    )} for ${getFormattedName(property.name)}`;
-    await createNotification(NOTIFICATIONS.OFFLINE_PAYMENT_ADDED, offlinePayment.userId, {
-      description,
-      actionId: payment._id,
-    });
+
+    if (model.propertyId) {
```

Comment:
I think it would be better to check the `modelType` instead of propertyId.

```suggestion
    if (modelType === MODEL.OFFER) {
```

Reply:
Also, it is better to generate the description and use just one await `createNotification(...` instead of including it in the `if` and `else` block.